openerp/addons/openupgrade_records/lib/apriori.py

```diff
@@ -29,6 +29,7 @@
     'marketing_security_group': 'mass_mailing_security_group',
     # odoomrp/odoomrp-wip
     'product_variants_no_automatic_creation': 'product_variant_configurator',
+    'sale_documents_comments': 'sale_comment_propagation',
```

Comment:
We know that. This is for ensuring the migration path from odoomrp-wip module to OCA. If you have already 8.0 OCA module, this is harmless (well, not yet).